app/scripts/modules/amazon/src/loadBalancer/configure/application/ALBAdvancedSettings.tsx

```diff
@@ -1,25 +1,40 @@
 import React from 'react';
 
+import { FormikProps } from 'formik';
 import { FormikFormField, CheckboxInput, NumberInput, HelpField } from '@spinnaker/core';
+import { IAmazonApplicationLoadBalancerUpsertCommand } from 'amazon/domain';
 
-export class ALBAdvancedSettings extends React.Component {
-  public render() {
-    return (
-      <div>
-        <FormikFormField
-          name="idleTimeout"
-          label="Idle Timeout"
-          help={<HelpField id="loadBalancer.advancedSettings.idleTimeout" />}
-          input={(props) => <NumberInput {...props} min={0} />}
-        />
+export interface IALBAdvancedSettingsProps {
+  formik: FormikProps<IAmazonApplicationLoadBalancerUpsertCommand>;
+}
+
+export const ALBAdvancedSettings = React.forwardRef<HTMLDivElement, IALBAdvancedSettingsProps>((props, ref) => (
+  <div ref={ref}>
+    <FormikFormField
+      name="idleTimeout"
+      label="Idle Timeout"
+      help={<HelpField id="loadBalancer.advancedSettings.idleTimeout" />}
+      input={(props) => <NumberInput {...props} min={0} />}
```

Comment:
I noticed you used `inputProps` down below to avoid shadowing the `props` variable.  Might be good to do that everywhere.  I thought we had a linter rule enabled that forbids shadowing?